packages/libllvm/build.sh

```diff
@@ -93,6 +96,8 @@ termux_step_pre_configure() {
 
 termux_step_post_configure() {
 	TERMUX_PKG_SRCDIR=$TERMUX_SRCDIR_SAVE
+	# add hostbuild dir to path so mlir-tblgen is found
+	export PATH=$TERMUX_PKG_HOSTBUILD_DIR/bin:$PATH
```

Comment:
No longer needed if you're patching the path in below?

Reply:
It seem to be needed, for whatever reason, även with the patch it runs just `mlir-tblgen` from PATH